src/Function/Bundles.agda

```diff
@@ -420,3 +421,55 @@ module _ {A : Set a} {B : Set b} where
   -- Sometimes the implicit arguments above cannot be inferred
   mk↔′ : ∀ (f : A → B) (f⁻¹ : B → A) → Inverseˡ f f⁻¹ → Inverseʳ f f⁻¹ → A ↔ B
   mk↔′ f f⁻¹ invˡ invʳ = mk↔ {f = f} {f⁻¹ = f⁻¹} (invˡ , invʳ)
+
+------------------------------------------------------------------------
+-- Conversion functions
+
+module _ (A : Setoid a ℓ₁) (B : Setoid b ℓ₂) where
+
+  Inverse⇒Bijection : Inverse A B → Bijection A B
+  Inverse⇒Bijection record { f = f ; f⁻¹ = f⁻¹ ; cong₁ = cong₁ ; cong₂ = cong₂ ; inverse = inverse } = record
+    { f =         f
+    ; cong =      cong₁
+    ; bijective = (λ {x} {y} x≈y → begin
```

Comment:
The proofs of the relationship between the individual properties (e.g. `Bijective`, `Inverseᵇ`) rather than bundles (i.e. `Bijection`, `Inverse`) should all live in `Function.Consequences` (doesn't currently exist).

Reply:
I'll move this one, but I don't think I can move the proof in `Bijection⇒Inverse` right now without a bunch of extra work. Doing this direction cleanly would probably require a version of `Inverseᵇ` for setoid morphisms.